### PR TITLE
JmaCsvCatalog class for reading JMA csv catalog files

### PR DESCRIPTION
**Pull request by thbeutin**
class JmaCsvCatalog(BaseCatalog) added. It reads a perl-created csv file.

### CODE REVIEW

File: csep/core/catalogs.py [hunk 1]
```diff
@@ -1036,3 +1036,121 @@ def get_csep_format(self):
 
         return CSEPCatalog(catalog=csep_catalog, catalog_id=self.catalog_id, filename=self.filename)
 
+
+class JmaCsvCatalog(BaseCatalog):
```
Comment (wsavran) [inline csep/core/catalogs.py]: AbstractBaseCatalog

File: csep/core/catalogs.py [hunk 2]
```diff
@@ -1036,3 +1036,121 @@ def get_csep_format(self):
...
+        ('longitude', 'f8'),
+        ('latitude', 'f8'),
+        ('depth', 'f8'),
+        ('magnitude', 'f8')
+    ])
+
+    def __init__(self, catalog_id='JMA', format='csv', **kwargs):
+        # initialize parent constructor
+        super().__init__(**kwargs)
+
```
Comment (wsavran) [inline csep/core/catalogs.py]: need to call load_catalog()

Comment (thbeutin) [reply csep/core/catalogs.py]: Didn't you requested lazy loading?

Comment (wsavran) [reply csep/core/catalogs.py]: yes, you're right.

File: csep/core/catalogs.py (hunk 1 above)
Comment (thbeutin) [reply csep/core/catalogs.py]: done.

Comment (thbeutin) [reply csep/core/catalogs.py]: > done.

(at least should be...)

**Pull request merged**